README.md

```diff
@@ -12,7 +12,50 @@ If you are new to the Windows Package Manager, you might want to [Explore the Wi
 
 ### Microsoft Store [Recommended]
 
-The client is distributed within the [App Installer](https://www.microsoft.com/p/app-installer/9nblggh4nns1) package. 
+The client is distributed within the [App Installer](https://www.microsoft.com/p/app-installer/9nblggh4nns1) package.
+
+### Microsoft Store using PowerShell
+
+```PowerShell
+$applicationId = "9NBLGGH4NNS1" # Microsoft.DesktopAppInstaller_8wekyb3d8bbwe
+$skuId = 0016
+
+# Obtain packageFamilyName from applicationId
+$webpage = Invoke-WebRequest -UseBasicParsing -Uri "https://bspmts.mp.microsoft.com/v1/public/catalog/Retail/Products/$applicationId/applockerdata"
+$packageFamilyName = ($webpage | ConvertFrom-JSON).packageFamilyName
```

Comment:
`Invoke-RestMethod` will automatically parse the JSON rather than requiring an explicit call to `ConvertFrom-JSON`